.github/coderabbit.yaml

```
@@ -49,8 +49,8 @@ checks:
 
 # Exclude specific files or directories
 exclude_patterns:
-  - "*.md"
-  - "LICENSE"
-  - "node_modules/**"
-  - "dist/**"
-  - "build/**"
\ No newline at end of file
+   - "*.md"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 3

(indentation)